mono/metadata/sgen-mono.c

```diff
@@ -2799,18 +2800,19 @@ mono_gc_add_memory_pressure (gint64 value)
  */
 
 void
-sgen_client_degraded_allocation (size_t size)
+sgen_client_degraded_allocation (void)
```

Comment:
I guess @schani would know. He did the whole SGen separation effort for a use case I can't remember.